docs/faq.md

```diff
@@ -33,6 +33,12 @@ include the `pnpm-workspace.yaml` if it exists and all `package.json` files in y
 To list all local `package.json` files that pnpm needs to read, you can run
 `pnpm recursive ls --depth -1 --porcelain`.
 
+## An npm package is failing when writing to the `node_modules` directory
```

Comment:
The issue that this answer originated from was a js_binary tool that was trying to write to a node_modules directory and not an npm package. https://github.com/aspect-build/rules_js/issues/279

I think the question is more general:

```
## Can a tool run outside of Bazel write to the `node_modules` in `bazel-out`?

Some tools such as the AWS SDK write to node_modules when they are run. Ideally this should be avoided or fixed upstream. ...
```
```

Reply:
Updated